frontend/packages/integration-tests-cypress/tests/monitoring/monitoring.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { timeouts } from '../../utils/timeouts';
+import { testName } from '../../utils/common';
+import { submitButton, errorMessage } from '../../utils/selectors';
+import { listPage } from '../../views/list-page';
+import { detailsPage } from '../../views/details-page';
+import { modal } from '../../views/modal';
+
+const shouldBeWatchdogAlertDetailsPage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Alert Details');
+  detailsPage.labelShouldExist('alertname=Watchdog');
+};
+
+const shouldBeWatchdogAlertRulesPage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Alerting Rule Details');
+  detailsPage.sectionHeaderShouldExist('Active Alerts');
+};
+
+const shouldBeWatchdogSilencePage = () => {
+  detailsPage.titleShouldBe('Watchdog');
+  detailsPage.sectionHeaderShouldExist('Silence Details');
+  detailsPage.labelShouldExist('alertname=Watchdog');
+};
+
+before(() => {
+  cy.login();
+  cy.createProject(testName);
+});
+after(() => {
+  cy.deleteProject(testName);
+  cy.logout();
+});
+
+describe('Monitoring: Alerts', () => {
```

Comment:
Remove the corresponding Protractor tests if we've moved these to Cypress.

Reply:
Well, I had converted it all, but @kyoto recently added `await testSilenceTimeInputs();` to it :-)

Reply:
> I had converted it all, but @kyoto recently added `await testSilenceTimeInputs();` to it 😉

Can we migrate that over as well? I don't think we should have two copies of these tests running.